### ISSUE TITLE
Unable to get it working, Would love some configuration instructions.

### ISSUE THREAD

**Issue by Sceptorrh** [Olen/home-assistant-openplantbook#2]
Hey,

So, I followed all the instructions up to the point where I have everything installed in homeassistant. I feel just the last part of the instructions is missing. It looks really promisting and I would really love to get it all working =)

Any idea on when te documentation will be finished?

Thanks for all the hard work!

**Comment by Olen** [comment]
I don't understand what you miss. Does the service calls not work? Have you added your api keys etc?

**Comment by itn3rd77** [comment]
Hey,

sorry for hijacking this issue but maybe @Sceptorrh you are hit by the same problem.

I have followed the instructions from the README but I can not add "OpenPlantbook" via "Add integration". It's simply not there. The two service calls are also missing.

Integration is installed and does exist at custom_components/

![grafik](https://user-images.githubusercontent.com/2655169/190576101-fbf59ce4-de95-44e9-97d5-e95d699a117c.png)


![grafik](https://user-images.githubusercontent.com/2655169/190575423-a41973bf-f939-41ae-af1d-0f17f30964b9.png)

Removed everything and reinstalled but can't get it working. I can't find any errors in the home-assistant.log. During start it recognized the custom integration and that's it.

![grafik](https://user-images.githubusercontent.com/2655169/190575981-1c482583-0cbc-4a19-a709-825185f413fb.png)
